en/orm/query-builder.rst

```diff
@@ -464,7 +464,7 @@ If we wanted to classify cities into SMALL, MEDIUM, or LARGE based on population
 size, we could do the following::
 
     $query = $cities->find()
-        ->where(function ($exp, $q) {
+        ->where(function (\Cake\Database\Expression\QueryExpression $exp, \Cake\ORM\Query $q) {
```

Comment:
Any chance we could use only the class name? The full name is a bit wordy.

Reply:
With a use statement above then?

Reply:
I agree it is long and tiresome on my eyes. I decided not to because in other places on this page we use the FQCN. 

I could see doing that if we added an anonymous function definition someplace that has the FQCN and then use the shortened version else where?  Would that work for you?